### PR TITLE
C24_WMDE_Desktop_EN_02 (round 1)

### PR DESCRIPTION
**Pull request by moiikana**
- based on en 01
- var: text change
- round 1, went live on 15th of October

- round 2 of this banner: https://github.com/wmde/fundraising-banners/pull/582

Ticket: https://phabricator.wikimedia.org/T374101

### CODE REVIEW

File: banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// This is the file where we import the theme-specific component styles
+@use 'src/themes/Treedip/swatches/skin_vector-2022' with (
```
Comment (Abban) [inline banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss:2]: When https://github.com/wmde/fundraising-banners/pull/579 is merged the payment icons here will break. You should be able to add `$select-group-image-label: true,` to the list already to stop that.

**Review by Abban — APPROVED** [review]

**Comment by moiikana** [comment]
rebased main for "is late progress" feature

File: banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss (hunk 1 above)
Comment (moiikana) [reply banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss:2]: added that line

Comment (Abban) [reply banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss:2]: Oh, I expected SASS would ignore an undefined parameter as the IDE doesn't throw warnings, but apparently the compiler doesn't like it.

Comment (moiikana) [reply banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss:2]: removed the line and will add it back later, when #579 is actually merged

**Comment by moiikana** [comment]
contains the latest UoF changes from https://github.com/wmde/fundraising-banners/pull/563 :heavy_check_mark: 
- [x] banner ctrl
- [x] banner var
- [x] fallback banner

**Comment by Sperling-0** [comment]
Already done:

⚠️ Rebase this PR from latest main.

To get the latest UoF working:

- For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
- For Mobile/ Ipad banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

**Comment by moiikana** [comment]
is rebased and contains latest changes necessary for https://github.com/wmde/fundraising-banners/pull/579

**Pull request merged**